### PR TITLE
Constitutions article

### PR DESCRIPTION
**Pull request by JackuB**
https://github.com/apiaryio/apiaryio.github.com/blob/JackuB/Constitutions/_posts/2018-04-25-Constitutions.md

### CODE REVIEW

File: _posts/2018-04-25-Constitutions.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[![Apiary Constitutions repository](/images/2018-04-25-Constitutions/constitutions-repo.png)](https://github.com/apiaryio/constitutions)
+
+It’s a repository prepared for the rules development. Comes with a default set of rules you have enabled when you start using Style Guide in Apiary. You can easily reuse them or make them your starting point for custom rules. Our goal was to supply you with a way to have a simple `build` and `test` commands.
+
+Style Guides functionality is available to Apiary Pro customers.
+
+### How does it work?
+To stay true to the example driven development, Constitutions will require of you to supply **both passing and failing examples** for your Style Guide rule. So you can see how a correct usage looks like. This also helps you test the rules as you develop them. These examples are then reused for generating a Readme, which should serve as a readable API Style Guide reference.
```
[the remark this answers is not in the document]
Comment (honzajavorek) [reply _posts/2018-04-25-Constitutions.md]: Readme has unusual casing (readme, README)

File: _posts/2018-04-25-Constitutions.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+We then use [Webpack](https://webpack.js.org) to build a single file with all your rules, which is then run through our rules engine. We recommend to fork the Constitutions repo if you wish to have your own API Style Guides in git.
+
+![Apiary Style Guide](/images/2018-04-25-Constitutions/apiary-style-guide.png)
+
+## Updates to the Apiary CLI
+Constitutions will simplify writing the rules and to allow you better integration of these rules into your API Design flow, we also added [`styleguide`](https://help.apiary.io/tools/apiary-cli/#using-apiary-style-guide) option to the [Apiary CLI](https://github.com/apiaryio/apiary-client). With it, you can fetch and test your Apiary Style Guides against a local document.
```
Comment (honzajavorek) [inline _posts/2018-04-25-Constitutions.md]: I'd split the sentences:

> Constitutions will simplify writing the rules. To allow you better integration of these rules...

Comment (honzajavorek) [reply _posts/2018-04-25-Constitutions.md]: missing article?

```diff
- also added `styleguide` option
+ also added the `styleguide` option
```

File: _posts/2018-04-25-Constitutions.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+[![Apiary Constitutions repository](/images/2018-04-25-Constitutions/constitutions-repo.png)](https://github.com/apiaryio/constitutions)
+
+It’s a repository prepared for the rules development. Comes with a default set of rules you have enabled when you start using Style Guide in Apiary. You can easily reuse them or make them your starting point for custom rules. Our goal was to supply you with a way to have a simple `build` and `test` commands.
+
+Style Guides functionality is available to Apiary Pro customers.
+
+### How does it work?
+To stay true to the example driven development, Constitutions will require of you to supply **both passing and failing examples** for your Style Guide rule. So you can see how a correct usage looks like. This also helps you test the rules as you develop them. These examples are then reused for generating a Readme, which should serve as a readable API Style Guide reference.
+
+We then use [Webpack](https://webpack.js.org) to build a single file with all your rules, which is then run through our rules engine. We recommend to fork the Constitutions repo if you wish to have your own API Style Guides in git.
```
[the remark this answers is not in the document]
Comment (JackuB) [reply _posts/2018-04-25-Constitutions.md]: I'm thinking it's helping there

**Comment by JackuB** [comment]
Thanks for the review, addressed comments

**Comment by netmilk** [comment]
@JackuB I like it. The only thing I'm missing is either a tl;dr on top or a micro paragraph framing it with "How to write API Style Guide rules collaboratively" I'd make that may be a title of the blogpost.

**Pull request merged**